usaspending_api/references/v2/views/location_autocomplete.py

```diff
@@ -192,26 +182,32 @@ def _format_county_results(self, es_results: List[Hit]) -> Union[List, None]:
         """Format Elasticsearch results containing county matches
 
         Args:
-            es_results: Elasticsearch results that contained a match in the `counties` field
+            es_results: Elasticsearch results that contained a match in the `counties.name` or `counties.fips` field
 
         Returns:
-            A list containing all county names that matched the `search_text` along with the
+            A list of objects with county_fips and county_name properties that matched the `search_text` along with the
             state_name and country_name fields.
 
         Example:
             [
-                {"county_name": "GADSDEN", "state_name": "FLORIDA", "country_name": "UNITED STATES"},
-                {"county_name": "CAMDEN", "state_name": "GEORGIA", "country_name": "UNITED STATES"}
+                {"county_fips": "12039", "county_name": "GADSDEN", "state_name": "FLORIDA", "country_name": "UNITED STATES"},
+                {"county_fips": "13039", "county_name": "CAMDEN", "state_name": "GEORGIA", "country_name": "UNITED STATES"}
             ]
         """
-
         if len(es_results) > 0:
-            counties = []
-            for doc in es_results:
-                for county in doc.meta.highlight.counties:
-                    counties.append(
-                        {"county_name": county, "state_name": doc.state_name, "country_name": doc.country_name}
-                    )
+            counties = [
```

Comment:
This seems to work exactly as expected. With all of the nested looped though we should make sure to validate against large QAT dataset before marking the ticket done. Not going to block on this though.